### DISCUSSION TITLE
noindex tag when using hugo-clarity theme

### DISCUSSION THREAD

**Discussion by gitorko** [chipzoller/hugo-clarity#376] [Q&A]
The theme seems to be adding a tag 
`<meta name="robots" content="noindex">`
which is causing google to skip the indexing. How can this be removed?

Similar question:
https://discourse.gohugo.io/t/unwanted-noindex-tag-being-inserted-into-every-page/26705

**Comment by onweru** [comment]
@gitorko, clearly this theme isn't adding the noindex line via the stated baseof.html file. https://github.com/chipzoller/hugo-clarity/blob/master/layouts/_default/baseof.html

**Reply by gitorko** [reply]
I was using aliases, the alias page was having `<meta name="robots" content="noindex">` but the actual post didnt have this tag. Which makes sense, actual post was getting indexed and the alias was not. So looks like its working correctly.
Google search manager was trying to index the alias as well hence got the error "Excluded by ‘noindex’ tag"
